velox/docs/configs.rst

```diff
@@ -432,6 +432,7 @@ Each query can override the config by setting corresponding query session proper
      - false
      - True if reading the source file column names as lower case, and planner should guarantee
        the input column name and filter is also lower case to achive case-insensitive read.
+       True if extracting the partition path as lower case.
```

Comment:
Is this accurate? How would users understand what this means?

My impression was that this flag affects the partition directory path created on table write, no?

Reply:
@mbasmanova Good suggestion. Updated.